cob_bringup/robots/raw3-3.xml

```diff
@@ -63,6 +63,9 @@
 		<include file="$(find cob_bringup)/tools/base_collision_observer.launch" >
 			<arg name="robot" value="$(arg robot)" />
 		</include>
+
+		<include file="$(find rplidar_ros)/launch/rplidar.launch" />
```

Comment:
Is ```rplidar_ros``` a publically available ROS-package?
If so, you can add a run_dependency to the package.xml.